sensor/common/centralclient/testdata/update-testdata.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# This script will update the testdata for the TLSChallenge tests.
+# Usage: ./update-testdata.sh
+#
+# Prerequisites:
+# - roxcurl
+# - deployed StackRox instance via ./deploy/k8s/deploy-local.sh
```

Comment:
why do we need an instance to create keys? how central key is created maybe we can use the same method here?

Reply:
Mostly it is not for generating the certificates in Central, I've done it to get the TLSChallenge response. I've not added the Central generation to the script, but called the TLSChallenge directly in a go script to receive the response.